services/ui-src/src/verbiage/pages/accordion.ts

```diff
@@ -85,7 +85,7 @@ export default {
           props: {
             href: "https://www.ecfr.gov/current/title-42/chapter-IV/subchapter-C/part-438/subpart-A/section-438.8#p-438.8(k)",
             target: "_blank",
-            ariaLabel: "Link opens in new tab",
+            "aria-label": "Link opens in new tab",
```

Comment:
I _think_ for these labels you want to include the visible text as well. 
`"aria-label": "Help File (link opens in new tab)"`
etc. 
When I tried this out with VoiceOver it only read the aria label (Link opens in new tab) and not the text (Help File), so I think the label needs both pieces of info. Maybe a different screen reader had a different result though?

Reply:
ohh, nice catch! 🌿

I tested this on Google Chrome using [this Screen Reader](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn/related) and it does indeed just say "Link opens in new tab" when clicking "User Guide" or "Help File". I will add another update that includes the visible text!